test/JDBC/expected/db_collation/babel_collection.out

```diff
@@ -184,13 +184,13 @@ GO
 text
 Query Text: select c1 from testing4 where c1 LIKE 'jones'
 Bitmap Heap Scan on testing4  (cost=11.40..31.15 rows=3 width=32)
-  Filter: ((remove_accents_internal((c1)::text))::text = 'jones'::text)
+  Filter: (remove_accents_internal((c1)::text) = 'jones'::nvarchar)
```

Comment:
shouldn't we get COLLATE on right operand? Seems like it should be just by looking at look

Reply:
No, as the the column does NOT have explicit collation (column level collation), it picks up the database default. As this is exact match, we use that collation for ```=``` operator as well. If the query would have been something like this:
```
select c1 from testing4 where c1 LIKE 'jones' COLLATE DATABASE_DEFAULT
```
then we would expect the COLLATE in plan